packages/auth/src/AuthConfirmSignUp/AuthConfirmSignUp.tsx

```diff
@@ -27,10 +27,8 @@ const AuthConfirmSignUp = ({
     <form
       onSubmit={handleSubmit(({ code }) => onConfirmSignUp({ code, email }))}
     >
-      <AuthCard title="Confirm SignUp" buttonLabel="Confirmar">
-        <FormField label="Confirmation Code">
-          <Input id="email" {...register('code')} />
-        </FormField>
+      <AuthCard buttonLabel="Confirmar">
+        <Input variant="forms.auth.input" id="email" {...register('code')} />
```

Comment:
We don't need to apply variants here since we're using the default input. The same is true for other inputs.